ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -430,84 +500,106 @@ class TreegraphSeries extends OrganizationSeries {
      * @private
      */
     public pointAttribs(
-        point: TreegraphPoint | TreegraphNode,
-        state: StatesOptionsKey
+        point: TreegraphPoint,
+        state?: StatesOptionsKey
     ): SVGAttributes {
-        if (point && point.isNode) {
-            const { opacity, ...attrs } = Series.prototype.pointAttribs.apply(
-                this,
-                arguments
+        const series = this,
+            attribs = Series.prototype.pointAttribs.call(series, point, state),
+            levelOptions =
+                (series.mapOptionsToLevel as any)[point.node.level || 0] || {},
+            options = point.options,
+            stateOptions =
+                (levelOptions.states &&
+                    (levelOptions.states as any)[state as any]) ||
+                {},
+            borderRadius = pick(
+                stateOptions.borderRadius,
+                options.borderRadius,
+                levelOptions.borderRadius,
+                series.options.borderRadius
+            ),
+            linkColor = pick(
+                stateOptions.link && stateOptions.link.color,
+                options.link && options.link.color,
+                levelOptions.link && levelOptions.link.color,
+                series.options.link && series.options.link.color
+            ),
+            linkLineWidth = pick(
+                stateOptions.link && stateOptions.link.lineWidth,
+                options.link && options.link.lineWidth,
+                levelOptions.link && levelOptions.link.lineWidth,
+                series.options.link && series.options.link.lineWidth
             );
-            return attrs;
+
+        if ((point as any).isLink) {
+            attribs.stroke = linkColor;
+            attribs['stroke-width'] = linkLineWidth;
+            delete attribs.fill;
+        } else {
+            if (borderRadius) {
+                attribs.r = borderRadius;
+            }
         }
-        return super.pointAttribs.apply(this, arguments);
+        return attribs;
     }
 
+    public drawPoints(): void {
+        super.drawPoints.apply(this, arguments);
+        const points = this.points;
+        this.points = this.links as any;
```

Comment:
I am not a fan of this casting. I was even wondering if we should not create a separate method for drawing links if they have different type and functionality, instead of doing this tricky solution. @pawelfus

Reply:
Cool idea, however, I'm not sure if this should be part of this PR

Reply:
Because this is not the only series with this issue. If we consider all series, we may come up with a more universal solution. I simply don't want this PR to grow indefinitely, it is already hard to review all of the changes